README.rst

```diff
@@ -765,6 +768,9 @@ passed to the ``mkpkg`` util. By default Makd passes the following variables:
 ``version``
         package version number as defined by ``PKGVERSION``.
 
+``iteration``
+        package iteration as defined by ``PKGITERATION``.
+
```

Comment:
You need to change `mkpkg` to expose `PKGITERATION`, is not automagically ✨

Reply:
Ah.  I assumed it would be handled automatically by this call:

```mak
    $(call exec,PYTHONDONTWRITEBYTECODE=1 $(MAKD_PATH)/mkpkg \
        $(if $V,,-vv) -D-p"$P" -F "$(FPM)" \
        $(PKG_DEFAULTS) \
        -d suffix="$(PKG_SUFFIX)" -d version="$(PKGVERSION)" \
        -d builddir="$G" -d bindir="$B" \
        -d fullname="$*$(PKG_SUFFIX)" -d shortname="$*" \
        $<,$<,mkpkg)
```

... where the `$(PKG_DEFAULTS)` are included, and therefore I assumed `iteration` would be included.  What do I need to change in `mkpkg` ... ?  (I'm afraid I'm having some difficulty following the python.)

Reply:
That would pass `-i` to `fpm`, but the `.pkg` script will know nothing about it. To *define* variables for `.pkg` scripts, you need to pass: `-d iteration="$(PKGITERATION)"` or something.

Reply:
Arguments to `fpm` != arguments to `mkpkg`.